Src/BootCamp.Chapter/User.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public readonly struct User : IEquatable<User>
```

Comment:
It's not a user, it's credentials

Reply:
I need to practice my English more. It was not sounding right to me but could not find better naming at that time. Cheers!